Sources/Subs-Db-postgresql.php

```diff
@@ -227,7 +227,7 @@ function smf_db_replacement__callback($matches)
 		break;
 
 		case 'identifier':
-			return '"' . strtr($replacement, array('`' => '', '.' => '')) . '"';
+			return '"' . strtr($replacement, array('`' => '', '"."' => '')) . '"';
```

Comment:
Are you sure this is what you want? Don't you want to change the value, not the key in the replace.